spcs-three-tier/setup.sh

```diff
@@ -1,5 +1,4 @@
 set -e
-snow sql -f "prepare/spcs_setup.sql"
```

Comment:
We do not need this anymore. Please see https://snowflakecomputing.atlassian.net/browse/SNOW-1527720